src/mlpack/methods/ann/layer/isrlu.hpp

```diff
@@ -0,0 +1,181 @@
+/**
+ * @file isrlu.hpp
+ * @author Pranav Reddy P
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_ISRLU_HPP
+#define MLPACK_METHODS_ANN_LAYER_ISRLU_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The ISRLU function, also known as Inverse Square Root Linear Unit
+ * It has a negative value allowing it to push the mean unit activation closer
+ * to zero and bring the normal gradient closer to the unit natural gradient,
+ * ensuring a noise robust deactivation state, lessening the over fitting
+ * risk. Experiments have led to the conclusion that ISRLU leads to faster
+ * learning and better generalization on CNN's than ReLU.
+ *
+ * It is defined by
```

Comment:
Can you cite https://arxiv.org/abs/1710.09967 here as well, here is an example how the format looks like:

https://github.com/mlpack/mlpack/blob/fded9ee333bfbac179e9826bdfe4a3b093ef67f2/src/mlpack/methods/ann/layer/elu.hpp#L52-L62